<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsEditorClass</class>
 <widget class="QMainWindow" name="GraphicsEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphicsEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Canvas" name="canvas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="ColorLabel" name="labelBaseColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="ColorLabel" name="labelBackgroundColor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>491</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="colors"/>
   </widget>
   <widget class="QGroupBox" name="groupTools">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>151</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="buttonPencil">
       <property name="text">
        <string>Pencil</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="buttonBrush">
       <property name="text">
        <string>Brush</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="buttonLine">
       <property name="text">
        <string>Line</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="buttonRectangle">
       <property name="text">
        <string>Rectangle</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="buttonEllipse">
       <property name="text">
        <string>Ellipse</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="buttonAirbrush">
       <property name="text">
        <string>Airbrush</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="buttonFill">
       <property name="text">
        <string>Fill</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="buttonColorPicker">
       <property name="text">
        <string>Color picker</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="buttonEraser">
       <property name="text">
        <string>Eraser</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBrushPatterns">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Brush patterns</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="layoutBrushPatterns"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>151</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Size</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButton_5px">
        <property name="text">
         <string>5 px</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_2px">
        <property name="text">
         <string>2 px</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_1px">
        <property name="text">
         <string>1 px</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_3px">
        <property name="text">
         <string>3 px</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="radioButton_10px">
        <property name="text">
         <string>10 px</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="radioButton_custom">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>px</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>label</zorder>
   <zorder>labelBackgroundColor</zorder>
   <zorder>canvas</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>labelBaseColor</zorder>
   <zorder>groupTools</zorder>
   <zorder>groupBrushPatterns</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorLabel</class>
   <extends>QLabel</extends>
   <header>colorlabel.h</header>
  </customwidget>
  <customwidget>
   <class>Canvas</class>
   <extends>QLabel</extends>
   <header>canvas.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="GraphicsEditor.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
